docs/text-area.md

```diff
@@ -0,0 +1,168 @@
+# Textarea
+
+![SCR-20221116-gzi](https://user-images.githubusercontent.com/20360870/202104722-6d5cf79a-60a7-4a06-ac61-b92aa086bf4c.png)
+
+
+Adapt UI provides one theme for text area component with three styles and eight interaction states. Use this component to receive information from the user.
+
+#### Simple Usage
+
+```js
+import { TextArea } from "@adaptui/react-native-tailwind"
+export default function App() {
+  return (
+      <TextArea placeholder="Type Something...." />
+  )
+}
+```
+
+## Table of Contents
+
+  - [Themes](#themes)
+  - [Variant](#variant)
+  - [Size](#size)
+  - [Icon](#icon)
+  - [Interactions](#interactions)
+  - [Props](#props)
+## Themes
+
+Adapt UI provides only one `base` theme for TextArea.
+
+![SCR-20221116-gzi](https://user-images.githubusercontent.com/20360870/202104722-6d5cf79a-60a7-4a06-ac61-b92aa086bf4c.png)
+
+<details>
+
+<summary>
+  <b>Usage</b>
+</summary>
+
+### Usage
+
+```js
+import { TextArea, Slot, Icon } from "@adaptui/react-native-tailwind"
+export default function App() {
+  return (
+      <TextArea
+        placeholder="Type Something..."
+      />
+  )
+}
+
+```
+
+</details>
+
+## Variant
+
+Adapt UI provides four different input field components styles: `outline`,
+`subtle` and `underline`.
+
+You can use these various styled input field components based on the necessity
+and style you wish to provide to your design.
+
+![SCR-20221116-h9c](https://user-images.githubusercontent.com/20360870/202107264-835c7781-9990-435f-a02a-7a08b3e2f04f.png)
+
+<details>
+
+<summary>
+<b>Usage</b>
+</summary>
+
+### Usage
+
+```js
+import { TextArea } from "@adaptui/react-native-tailwind"
+
+export default function App() {
+  return (
+    <>
+      <TextArea placeholder="Type something" />
```

Comment:
Having just "Type something" everywhere seems boring. 
Try to have some real use cases in previews everywhere. @crstnmac

Reply:
ok